.github/workflows/ci.yml

```diff
@@ -100,7 +99,6 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
-          token: ${{ secrets.AXIS_NOW_ACCESS_TOKEN }}
       - id: nvmrc
         run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
```

Comment:
We need to sign up to see the reports, but it won't fail the build if we don't. We can decide later if we want to keep coveralls or not. It might be nice to put a coverage badge on the readme at some point.